### ISSUE TITLE
Change faction clarifications

### ISSUE THREAD

**Issue by Helias** [chromiecraft/chromiecraft#7482]
### Bug Title

change faction does not include enough details

### Current Behaviour

We should list what happens to all the reputations, items, quests etc. faction related.

### Expected Behaviour

More details in the production description

### Steps to Reproduce

.

### Browser

Chrome

### Device

Desktop

### Operating System

Linux

**Comment by TheSCREWEDSoftware** [comment]
I'm currently working in adding comment column per faction (alliance and horde) and their respective texts. 

While i'm waiting for more inside how to propely do this refering to this discord [messages](https://discord.com/channels/217589275766685707/689803159086170250/1304272913498902529) in AC

I've temporarly working on this PR https://github.com/azerothcore/azerothcore-wotlk/pull/20480 which could allow you to read the comment in formation of the database instead of using the function if it would prove more efficient.

(main reason i look at this tables was because of this issue which i saw yesterday)

**Comment by Nyeriah** [comment]
I don't think this barrage of information is either required or useful in the website page tbh. I would be more concerned about people being aware Faction Change includes services such as Customization, Race Change, Rename, etc.

Likewise there should be info informing you can't Race Change druids. Or maybe disable it for that class completely.

**Comment by heyitsbench** [comment]
Maybe a note encouraging people to copy their character to the PTR and try a faction change there before using the service would be beneficial.

**Comment by Nyeriah** [comment]
People have difficulty reading long pages of text. I am afraid a long list of entries hardly few are concerned about will just clutter the page and for the most part be ignored.

**Comment by TheSCREWEDSoftware** [comment]
> I don't think this barrage of information is either required or useful in the website page tbh. I would be more concerned about people being aware Faction Change includes services such as Customization, Race Change, Rename, etc.
> 
> A note saying achievements, quests, items and reputations will be converted to their counterparts should be enough. Maybe mention the most frequently sought for ones as example. Have hardly seen anyone ask or complain about this though.
> 
> Likewise there should be info informing you can't Race Change druids. Or maybe disable it for that class completely.

To add to this example of Nyeriah, reflecting a recent issue ticket we got.

It seems if they have any pending auctions they will not be able to change faction. This user had no HD Models patch, clean version and already had cleared their mailbox (they got that mesasge before and they went to clear it)

![image](https://github.com/user-attachments/assets/5764120c-7bff-4c75-aefe-a09c517ed2c1)


![image](https://github.com/user-attachments/assets/e637da46-27a1-4fca-909f-c3533e9e8007)
